src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -139,7 +160,10 @@ namespace AppInstaller::CLI::Execution
     // arguments via Execution::Args.
     struct Context
     {
-        Context(std::ostream& out, std::istream& in) : Reporter(out, in) {}
+        Context(std::ostream& out, std::istream& in) : Reporter(out, in) { m_data[Data::ContextFlag] = ContextFlag::None; }

```

Comment:
>m_data[Data::ContextFlag] = ContextFlag::None; [](start = 75, length = 46)

Use Add

Reply:
Actually, I think having GetOrAdd would be better

---
In reply to: [503506625](https://github.com/microsoft/winget-cli/pull/588#discussion_r503506625) [](ancestors = 503506625)